.github/workflows/release.yml

```diff
@@ -55,12 +55,12 @@ jobs:
           echo "filename=${ZIPFILE}" >> $GITHUB_OUTPUT
 
       - name: Adjust release tag
-        uses: EndBug/latest-tag@v1.6.1
+        uses: EndBug/latest-tag@v1.6.2
         with:
           ref: "v${{ steps.rel_ver.outputs.ver }}"
 
       - name: Create release draft
-        uses: softprops/action-gh-release@v0.1.15
+        uses: softprops/action-gh-release@2
```

Comment:
uses: softprops/action-gh-release@v2.0.4

Reply:
Interesting. Some days ago when I released Nerd Fonts that workflow there with the same line did work.
![image](https://github.com/lukas-w/font-logos/assets/16012374/11391f12-5387-43f8-a869-c4ddb8ffcc40)

And they have it like this on their manual page:
![image](https://github.com/lukas-w/font-logos/assets/16012374/9835fdcb-144f-4b73-b0c3-c3919d33000e)

Compare also
https://www.github.com/softprops/action-gh-release/pull/406#issuecomment-1986390444

@zen0bit Are you sure it throws that error?

Reply:
ah I missed the `v`

I would never tack it to a concrete release (2.0.4) instead of a major version (2)

Thanks, fixing.